### ISSUE TITLE
Slider Accessibility Issue

### ISSUE THREAD

**Issue by bahri-dev** [microsoft/react-native-windows#11476]
### Problem Description

Hi,
I tried to make a View component like the following.

```js
<View
accessibilityLabel={'React Native View as Slider'}
accessibilityRole="adjustable"
focusable
accessible
accessibilityValue={{
min: 1,
max: 100,
now: 20,
text: 20,
}}
/>
```

Narrator reads the value of the above View component as "zero percent". What I expected, it needs to read "20 percent".
How can fix it?

Thanks,

### Steps To Reproduce

It's a simple UWP with a View component.

### Expected Results

Narrator should read the value of Slider correctly.

### CLI version

0.71.0

### Environment

```markdown
info Fetching system and libraries information...
System:
OS: Windows 10 10.0.19044
CPU: (16) x64 11th Gen Intel(R) Core(TM) i7-11800H @ 2.30GHz
Memory: 2.88 GB / 15.67 GB
Binaries:
Node: 16.13.1 - C:\Program Files\nodejs\node.EXE
Yarn: 1.22.19 - ~\AppData\Roaming\npm\yarn.CMD
npm: 8.1.2 - C:\Program Files\nodejs\npm.CMD
Watchman: Not Found
SDKs:
Android SDK: Not Found
Windows SDK:
AllowDevelopmentWithoutDevLicense: Enabled
AllowAllTrustedApps: Enabled
Versions: 10.0.16299.0, 10.0.17134.0, 10.0.17763.0, 10.0.18362.0, 10.0.19041.0, 10.0.22000.0
IDEs:
Android Studio: Not Found
Visual Studio: 16.11.33423.256 (Visual Studio Enterprise 2019)
Languages:
Java: 1.8.0_302 - C:\Program Files\Eclipse Foundation\jdk-8.0.302.8-hotspot\bin\javac.EXE
npmPackages:
@react-native-community/cli: Not Found
react: 18.2.0 => 18.2.0
react-native: 0.71.0 => 0.71.0
react-native-windows: 0.71.4 => 0.71.4
npmGlobalPackages:
react-native: Not Found
```

### Target Platform Version

None

### Visual Studio Version

None

### Build Configuration

None

### Snack, code example, screenshot, or link to a repository

```js
import React from 'react';
import {View} from 'react-native';

const App = () => {
return (
<View
accessibilityLabel={'React Native View as Slider'}
accessibilityRole="adjustable"
focusable
accessible
accessibilityValue={{
min: 1,
max: 100,
now: 20,
text: 20,
}}
/>
);
};

export default App;
```

**Comment by YajurG** [comment]
So I tried with the following code sample, and Narrator was reading 20% when I tabbed onto the View - just changed the `min` prop to 0 and the `text` prop to a string (you had it as an int and it was highlighting as a syntax error). With the `min` prop set to 1, it was reading out 19% for me.

```
<View
    accessibilityLabel={'React Native View as Slider'}
    accessibilityRole="adjustable"
    focusable
    accessible
    accessibilityValue={{
      min: 0,
      max: 100,
      now: 20,
      text: "20",
    }}>
</View>
```

**Comment by bahri-dev** [comment]
> So I tried with the following code sample, and Narrator was reading 20% when I tabbed onto the View - just changed the `min` prop to 0 and the `text` prop to a string (you had it as an int and it was highlighting as a syntax error). With the `min` prop set to 1, it was reading out 19% for me.
> 

Did you test the UWP? Does windows narrator read it correctly?

**Comment by YajurG** [comment]
> Did you test the UWP? Does windows narrator read it correctly?

Yes I tested it with Narrator and it read out 20% when I focused onto the View. By testing the UWP do you mean looking at the UIA tree through Accessibility Insights?

**Comment by bahri-dev** [comment]
> By testing the UWP do you mean looking at the UIA tree through Accessibility Insights?

No, I mean windows application is based on windows 10. I tried to run the sample application with only the above View component in windows 10, and Narrator reads "zero percent" for that.

**Comment by YajurG** [comment]
> No, I mean windows application is based on windows 10. I tried to run the sample application with only the above View component in windows 10, and Narrator reads "zero percent" for that.

Did you try using my code sample and seeing what Narrator read out? If there was a change?

**Comment by bahri-dev** [comment]
> Did you try using my code sample and seeing what Narrator read out? If there was a change?

The difference is text is string. Am I right? I tried the test as string, but the same error.

**Comment by YajurG** [comment]
Also changed the `min` in `accessibilityValue` to 0 but not sure if that will make a difference if you're getting 0 read out by narrator.

**Comment by bahri-dev** [comment]
@YajurG , could you share your repo? I'll pull and test your app in my side.

**Comment by YajurG** [comment]
https://github.com/YajurG/react-native-windows

I tested on this branch

**Comment by bahri-dev** [comment]
> https://github.com/YajurG/react-native-windows

How can I run the app in windows?

**Comment by YajurG** [comment]
You can just start Playground and then use one of the sample pages. There should be instructions for this in the README in the `packages/playground` directory.

**Comment by bahri-dev** [comment]
@YajurG ,  I tried to build playground app in Win 11, but it's failed.
![react-native-windows playground build error](https://github.com/microsoft/react-native-windows/assets/92716081/1a1e386f-8099-41f3-bf21-2865d2af9805)
Could you let me know how I can fix it?

**Comment by YajurG** [comment]
That's strange, it's building okay for me. Did you clone/pull from the main RNW repo?

**Comment by bahri-dev** [comment]
Yes, I cloned from it.

**Comment by bahri-dev** [comment]
After I cloned it, I ran "yarn install", "yarn build" in root path. And, "cd packages\playground". And, open playground-composition.sln in Visual Studio 2022. And run it in "Debug|x64" mode.

**Comment by YajurG** [comment]
Can you try running the playground.sln file and building that one?

**Comment by bahri-dev** [comment]
Yes, it's also failed.

**Comment by YajurG** [comment]
Going to close this issue since I cannot reproduce the error locally. Can you open up a separate issue detailing your issues with running playground with your react-native-windows version information?

**Comment by YajurG** [comment]
You can follow the steps here to run playground: https://github.com/microsoft/react-native-windows/tree/main/packages/playground
